src/staff/domain/policies/role.policy.ts

```
@@ -0,0 +1,98 @@
+    valid = this.validateNamePattern(name);
+    if (!valid.success) return valid;
+    valid = this.validateNameLength(name);
+    if (!valid.success) return valid;
+
+    return valid;
+  }
+
+  public static validateCode(
+    code: string,
+  ): Result<string, DomainValidationError> {
+    this.validateRequiredString(code, '역할 코드');
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)